todo_list/forms.py

```diff
@@ -0,0 +1,23 @@
+from django import forms
+
+from todo_list.models import Page, Task
+
+
+class PageForm(forms.ModelForm):
+    class Meta:
+        model = Page
+        fields = '__all__'
+
+
+class TaskForm(forms.ModelForm):
+    class Meta:
+        model = Task
+        fields = '__all__'
+
+    def __init__(self, *args, **kwargs):
+        page = kwargs.pop('page', '')
+        super(TaskForm, self).__init__(*args, **kwargs)
+        self.fields['is_complete'].required = False
```

Comment:
@quratulain25 Can we define these properties at form fields instead of updating those in __init__?